server.js

```diff
@@ -0,0 +1,77 @@
+
+let bodyParser = require('body-parser');
```

Comment:
This is just being nit-picky and doesn't affect the code or performance, but just as a convention, the bodyParser should come after express.